packages/zeplin-stylus-extension/package.json

```diff
@@ -11,6 +11,7 @@
     "publish": "zem publish"
   },
   "dependencies": {
+    "base-extension": "^1.0.0",
```

Comment:
Don't we need to publish base-extension?

Reply:
Nope, this is just an internal package used by actual stylesheet extensions. We don't intend to share this as an npm package.